src/elements/core/testing/wait-for-image-ready.ts

```
@@ -1,20 +1,41 @@
 import type { SbbImageElement } from '../../image.js';
+import { isSafari } from '../dom.js';
 
 export async function waitForImageReady(
   element: HTMLImageElement | SbbImageElement,
   timeoutInMilliseconds = 2 * 1000,
 ): Promise<void> {
+  const imgElement =
+    element.localName === 'sbb-image'
+      ? (element.shadowRoot?.querySelector<HTMLImageElement>('.sbb-image__img') ?? null)
+      : (element as HTMLImageElement);
+
+  if (!imgElement) {
+    return Promise.reject('img tag not found');
```

I think you can just throw an Error?